src/task-implementation.rst

```diff
@@ -50,22 +50,22 @@ independent testing, re-use, and shell mode editing.
 Task Messages
 -------------
 
-Tasks messages can be sent back to the suite server program to report completed
-outputs and arbitrary messages of different severity levels.
-
-Some types of message - in addition to events like task failure -  can
-optionally trigger execution of event handlers in the suite server program
-(see :ref:`EventHandling`).
+Task jobs automatically send status messages back to the server program to
+report that execution has started, succeeded, or failed. Custom messages
+can also be sent back by the same mechanism, with various severity levels.
+These can be used to trigger other tasks off specific task outputs, or 
+to trigger execution of event handlers by the suite server program (see
+:ref:`EventHandling`), or just to write information to the server log.
 
 Normal severity messages are printed to ``job.out`` and logged by the
-suite server program:
+server program:
```

Comment:
"suite" removed here but not in multiple places below. Probably not worth worrying about - we'll need a major review of the docs if we want to get rid of all references to "suite server".

Reply:
Yeah, I'm trying to avoid using "suite" in new documentation updates, but fairly half-heartedly because I'm aware we'll have to do a full sweep through all the docs before the 8.0.0 release.